R/g.loadlog.R

```diff
@@ -198,15 +200,18 @@ g.loadlog = function(loglocation = c(), coln1 = c(), colid = c(), nnights = c(),
       }
     }
   }
-  nnights = nnights + deltadate
+  nnights = nnights + deltadate + 1
```

Comment:
Maybe add code comments to explain why we do +1? Is this to account for possibility that there is an extra night at the beginning that needs to be accounted for?